main.js

```diff
@@ -0,0 +1,50 @@
+const form = document.querySelector('form');
+const cityInput = document.querySelector('#city');
+const submitBtn = document.querySelector('#submitBtn');
+const weatherData = document.querySelector('#weatherData');
+
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+
+  const city = cityInput.value.trim();
+
+  if (city) {
+    const API_KEY = '5d066958a60d315387d9492393935c19';
+    const url = `http://api.openweathermap.org/data/2.5/weather?q=${city}&units=metric&APPID=${API_KEY}`;

```

Comment:
Plz, check the comments from this MR

https://github.com/Kross55/YakushenkoFrontEndPro29.01.23/pull/11